src/screens/Profile/EditTariffPlanInformation.tsx

```diff
@@ -40,7 +43,7 @@ function EditTariffPlanInformation(props: EditTarifPlanInformationProps) {
         const freeUnits = parseFloat(newFreeUnits);
 
         if (isNaN(fixedTariff) || isNaN(tariffByUnit) || isNaN(freeUnits)) {
-            setErrorMessage('All fields must be filled with numbers');
+            setErrorMessage(intl(words.all_fields_must_be_filled_with_numbers));
```

Comment:
I'll be replacing this with `formik` validation instead of this ratchet error messaging soon (probably Friday 4/22). If we get this PR in before that, I can just do the translation stuff myself since I think I have an understanding of how the workflow is. I'll assign you to review the PR if that happens